bee-test/src/rand/mod.rs

```diff
@@ -23,16 +25,16 @@ pub mod number;
 pub mod option;
 /// Module providing random output generation utilities.
 pub mod output;
-/// Module providing random output diff generation utilities.
-pub mod output_diff;
+// Module providing random output diff generation utilities.
+// pub mod output_diff;
 /// Module providing random parents generation utilities.
 pub mod parents;
 /// Module providing random payload generation utilities.
 pub mod payload;
 /// Module providing random receipt generation utilities.
 pub mod receipt;
-/// Module providing random snapshot generation utilities.
-pub mod snapshot;
+// Module providing random snapshot generation utilities.
+// pub mod snapshot;
```

Comment:
reminder

Reply:
Can't be done as part of this PR without fixing the ledger crate